carapp/authent.py

```diff
@@ -0,0 +1,20 @@
+from carapp import auth, db_models, login_manager, token_serializer
+
+
+@login_manager.user_loader
+def user_loader(user_id):
+    user = db_models.User.query.get(user_id)
+    if not user:
+        return None
+
+    return user
```

Comment:
There is no need or even space for fancy moves.

```suggestion
    return user
```

Reply:
Stop, you mean like this?
```diff
+ user = db_models.User.query.get(user_id)
- if not user:
-        return None
+ return user
```

Reply:
Yeah, I accidentally included in the suggestion an extra line that must be preserved.

Reply:
Ok, yes!